kotlinx.fuzz.jazzer/src/main/kotlin/kotlinx/fuzz/jazzer/JazzerLauncher.kt

```diff
@@ -27,6 +29,22 @@ object JazzerLauncher {
     private val log = LoggerFacade.getLogger<JazzerLauncher>()
     private val config = KFuzzConfig.fromSystemProperties()
     private val jazzerConfig = JazzerConfig.fromSystemProperties()
+    private var oldRepresentatives: Int? = null
```

Comment:
Can you explain what is the purpose of this property?

Reply:
If user already have some crashes in the directory, we need to count them at the beginning so that our `keepGoing` will depend only on new findings. Aggeg a comment about that